setup.py

```diff
@@ -392,31 +392,30 @@ def has_function(self, function, includes=None, objects=None, libraries=None,
 
         msg = "Checking"
         if function:
-            msg += " for %s" % function
+            msg += f" for {function}"
         if includes:
             msg += " with includes " + str(includes)
         msg += "..."
         status = "no"
 
-        log.debug("objects: %s" % objects)
-        log.debug("libraries: %s" % libraries)
-        log.debug("include dirs: %s" % include_dirs)
+        log.debug(f"objects: {objects}")
+        log.debug(f"libraries: {libraries}")
```

Comment:
No need to change this here at all (as in, this is extraordinarily nitty), but perhaps you find this oddity interesting: https://docs.python.org/3/howto/logging.html#optimization

TL;DR: interpolation in logging statements is usually done with `'%s'` style and passing in arguments to the log function.

Again, only mentioned because I find the case interesting – no need to change here